### ISSUE TITLE
SmartStatus-OP weather 

### ISSUE THREAD

**Issue by kakarot1657** [robhh/SmartStatus-AppStore#5]
Hi I just Installed SmartStatus-OP on my pebble Running 2.0 beta 6 and I have been unable to get the Weather or calendar info to show up it just times out. Is there a new update for the smartwatch+ app on the iPhone to get it to display on the watch.

**Comment by robhh** [comment]
Yes, you can download the 2.0 versions of Smartwatch+ and SmartStatus here: http://developer.getpebble.com
